guide/14-deep-learning/how_sr3_works.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>See if you can put the two cases in bullet points. Will be clearer. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>